crates/ark/src/variables/variable.rs

```diff
@@ -327,6 +363,12 @@ impl WorkspaceVariableDisplayType {
     /// - include_length: Whether to include the length of the object in the
     ///   display type.
     pub fn from(value: SEXP, include_length: bool) -> Self {
+        match Self::try_from_method(value, include_length) {
+            Err(e) => log::error!("Error from 'ark_variable_display_type' method: {e}"),
```

Comment:
`positron_` or whatever we decide on

Reply:
Fixed in https://github.com/posit-dev/ark/pull/560/commits/fdee0f42f4727615f5f3b31a4aa41c061f849781